### DISCUSSION TITLE
Few seconds of video dropped

### DISCUSSION THREAD

**Discussion by eviljust** [streamlink/streamlink#6066] [Q&A]
### Description

When using the following command streamlink "m3u8 link" 1080p -o output.ts, only first few sec of the video is stopped but audio  heard. After like around 3 to 5 sec later the video resume.

Not sure what arg to add to not have the first few sec of dropped video frames. Or is it my machine or network?

Using streamlink version 6.7.0

Taken from the m3u8 manifest on the 1080p stream, not sure if this is any help
`#EXT-X-SESSION-DATA:DATA-ID="CUSTOMER_ID",VALUE="658583162646"
#EXT-X-SESSION-DATA:DATA-ID="CONTENT_ID",VALUE="o3SzXcbVZfOn"
#EXT-X-MEDIA:TYPE=VIDEO,GROUP-ID="chunked",NAME="1080p",AUTOSELECT=YES,DEFAULT=YES
#EXT-X-STREAM-INF:BANDWIDTH=9033584,RESOLUTION=1920x1080,CODECS="avc1.640028,mp4a.40.2",VIDEO="chunked",FRAME-RATE=30.000`

**Comment by DeltaEpsilon19498** [comment]
Can you give complete terminal output with the --loglevel=debug option?

What video player are you using? If you are using vlc, have you tried mpv instead? Have you tried using the --hls-live-edge option?
